modules/features/feature_inope_policy/feature_inope_policy.features.menu_links.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file
+ * feature_inope_policy.features.menu_links.inc
+ */
+
+/**
+ * Implements hook_menu_default_menu_links().
+ */
+function feature_inope_policy_menu_default_menu_links() {
+  $menu_links = array();
+
+  // Exported menu link: navigation_policy:node/add/inope-policy.
+  $menu_links['navigation_policy:node/add/inope-policy'] = array(
```

Comment:
Woah --- why are we exporting node add menu items -- with the feature enabled aren't these there automatically?